components/pixel/schema.json

```diff
@@ -0,0 +1,38 @@
+{
+  "title": "Pixel Component Definition",
+  "description": "Addressable Pixel WipperSnapper component for use in Adafruit IO",
+  "type": "object",
+  "required": [ "displayName", "pixelType"],
+  "properties": {
+    "displayName": {
+      "description": "The human-friendly name of this component.",
+      "type": "string",
+      "minLength": 3,
+      "maxLength": 24
+    },
+    "pixelType": {
+        "description": "The type of addressable pixel. 0 is unspecified, 1 is WS2812 (NeoPixel), 2 is APA201 (DotStar).",
+        "type": "number",
+        "minLength": 0,
+        "maxLength": 2
+    },
```

Comment:
Let's go with strings again here instead of magic numbers, so `neopixel` and `dotstar` instead of numbers that map to those meanings. Keeps it human-read/writeable for newer folks!

You can look to how I validated the sensor types with a big regex full of `|`s in other component types

Reply:
Switched to strings, using the same validation regex as the i2c component type.